Week2/eight.js

```diff
@@ -0,0 +1,75 @@
+// 8.1
+
+let myfavoriteanimals = ['lions', 'horses','eagles'];
```

Comment:
small remark, it's better to use `camelCase`, so then it's better to write `myFavouriteAnimals` instead of `myfavouriteanimals` because it's easier to read.
The same goes for your other variables

Reply:
Dear @marcianoviereck ;  just reviewed the freecodecamp basicjavascript part, and read camelCase there as weel , from now on I need to  pay attention to that.  So;
myfavoriteanimals should be myFavoriteAnimals; 
myfavoritetennisplayer should be myFavoriteTennisPlayer

thanks for the feedback

Reply:
Ah that's good!